gradle.properties

```diff
@@ -17,3 +17,13 @@ android.enableJetifier=false
 
 # Update Stripe.VERSION_NAME when publishing a new release
 VERSION_NAME=17.1.2
+
+
+# Set to example backend deployed to Heroku
+STRIPE_EXAMPLE_BACKEND_URL=https://stripe-testing-yp.herokuapp.com/
+
+# Set to a test publishable key from https://dashboard.stripe.com/test/apikeys
+STRIPE_EXAMPLE_PUBLISHABLE_KEY=pk_test_51JVgBBSCzPyv1Go9PIcd7l8F2lvNdydL9QOK9tpfdoKibi2OyhYPnvrciAzYfYmYhOAQ9qLnwGnVZTxlXroYg2Ew00HxYmTZwT
```

Comment:
Nit: we want to avoid committing these as everyone's are different.

Reply:
Okay, I'll remove this